content/en/docs/concepts/services-networking/service.md

```diff
@@ -94,12 +97,12 @@ spec:
       targetPort: 9376
 ```
 
-This specification creates a new Service object named "my-service", which
-targets TCP port 9376 on any Pod with the `app.kubernetes.io/name=MyApp` label.
+That manifest creates a new Service named "my-service", which
+targets TCP port 9376 on any Pod with the `app.kubernetes.io/name: MyApp` label.
 
-Kubernetes assigns this Service an IP address (sometimes called the "cluster IP"),
-which is used by the Service proxies
-(see [Virtual IP addressing mechanism](#virtual-ip-addressing-mechanism) below).
+Kubernetes assigns this Service an IP address (sometimes called the _cluster IP_)
```

Comment:
sometimes called? If the project explicitly calls it that, we can just define it as the canon term.

Reply:
I don't know the answer here.

Reply:
IDK the answer either but if this is the same cluster IP as the the clusterIp assigned by k8s then I think we're safe to just call it "assigns this Service an IP address, called the _cluster IP_."